CONTRIBUTING.md

```diff
@@ -0,0 +1,51 @@
+# Contributing to Archive of Our Own
+
+## Contributing code
+
+### Pick an issue to work on
+
+You will find our [list of approved issues on JIRA](https://otwarchive.atlassian.net/projects/AO3/issues).
+
+We regret that we cannot accept pull requests for issues we haven't already logged, with the exception of documentation 
+or spelling corrections. 
+
+### Resolve the issue
+
+Consult our [Developer Documentation](https://github.com/otwcode/otwarchive/wiki) on how to set up a local development 
+environment so you can work on the Archive code.
+
+### Submit your pull request
+
+Please read the checklist on [our pull request template](https://github.com/otwcode/otwarchive/blob/master/.github/PULL_REQUEST_TEMPLATE.md)
+on the steps you need to perform to make we can accept your pull request.
+
+### Action comments on your pull request
+
+When we review your pull request, we will make comments on Github. Please make sure you read these carefully and apply
+their suggestions!
+
+Due to our workload, it may take several weeks before an approved pull request is merged, so be patient - we will get to it!
+
+### Testing and fixing bugs
+
+Once your pull request is merged, it will be deployed to our internal Staging server and our manual QA team will check it.
+We may ask you to make further changes and submit a new pull request if we find any problems that will affect our users'
+ability to benefit from your changes. 
+
+### Release
+
+Once we have checked that all is well, your contribution will be deployed to [Archive of Our Own](http://archiveofourown.org) and you will be credited
+in the [AO3 release notes](http://archiveofourown.org/admin_posts?tag=1)! Make sure you tell us your preferred name and pronoun
```

Comment:
We capitalize Release Notes in the pull request template above (`What name do you want us to use to credit your work in the [Archive of Our Own's Release Notes](http://archiveofourown.org/admin_posts?tag=1)?`) -- should we do the same here?